### PR TITLE
test: increased async test timeout to 10s from 5s

### PR DESCRIPTION
**Pull request by Souvikns**
**Description**
The tests are failing sometimes, we don't know all the reasons the tests might be failing but recently we figured that some tests are failing because they are taking more time than the async timeout that was set to 5s. This PR aims to resolve these issues, currently, all the tests in the development server are passing so no way to understand which tests cases are taking more time than intended. 

It was running fine in ubuntu but was having issues in mac and windows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
     "pretest:coverage": "npm run build",
     "release": "semantic-release",
     "start": "npm run build && node dist/cli.js",
-    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 5000 \"test/**/*.test.ts\"",
+    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 6000 \"test/**/*.test.ts\"",
```
Comment (magicmatatjahu) [inline package.json:136]:
```suggestion
    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 10000 \"test/**/*.test.ts\"",
```

If you wanna increase timeout, you shouldn't increase it by 1s, but as doubled previous value, so 5s -> 10s. If it's really problem with timeout, 1s probably won't be a fix, better to have some "backup" of timeout :)

Comment (Souvikns) [reply package.json:136]: 😄 sure let's have some backup timeout.

**Review by magicmatatjahu — APPROVED** [review]
LGTM, but please wait for @derberg review :)

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
I think conventional commits have prefix for test but don't remember if it is `test` or tests`

**Comment by Souvikns** [comment]
Yeah it is `test` shall I update the pt title ?

**Comment by derberg** [comment]
@Souvikns yes please, or just update it in the commit message when you will squash and merge

**Pull request merged**